docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,11 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+DAML Compiler
+~~~~~~~~~~~~~
+
+- **BREAKING CHANGE**: Drop support for DAML-LF 1.3. Compiling to DAML-LF 1.4 should work without any code changes, although we highly recommend not specifying a target DAML-LF version at all.
```

Comment:
I thought I had written this before, but please note that older packages _will still work with the new server_, it's just that you cannot compile with an older target.

Reply:
Addressed.